bin.src/movingObjects.py

```diff
@@ -0,0 +1,1535 @@
+#!/usr/bin/env python
+
+from __future__ import print_function, division
+
+import os
+import argparse
+import numpy as np
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+import lsst.sims.maf.metrics as metrics
+import lsst.sims.maf.slicers as slicers
+import lsst.sims.maf.stackers as stackers
+import lsst.sims.maf.plots as plots
+import lsst.sims.maf.db as db
+import lsst.sims.maf.metricBundles as mmb
+import lsst.sims.maf.utils as utils
+
+
+# Assumes you have already created observation file,
```

Comment:
Would be great to put an example notebook or script in the examples folder (or sims_maf_contrib) that generates an observation file, and then runs the new metrics and document the output plots a bit. I'm just worried that this is generating a ton of cool stuff, but no one else will be able to interpret the output. Maybe a notebook that compares a pairs-on vs pairs-off run, so it's really obvious how things change?